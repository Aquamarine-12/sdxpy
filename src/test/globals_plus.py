my_variable = 123
print(globals())
