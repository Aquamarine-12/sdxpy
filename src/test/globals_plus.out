{'__name__': '__main__', '__doc__': None, '__package__': None, \
'__loader__': <_frozen_importlib_external.SourceFileLoader object at \
0x7f95f8194df0>, '__spec__': None, '__annotations__': {}, \
'__builtins__': <module 'builtins' (built-in)>, '__file__': \
'/sd4ds/test/globals_plus.py', '__cached__': None, 'my_variable': 123}
